src/web/server/htmlTemplate.ts

```diff
@@ -99,7 +99,12 @@ export const htmlTemplate = ({
 		`https://support.theguardian.com`,
 	];
 
-	const preconnectTags = staticPreconnectUrls.map(
+	const allStaticPreconnectUrls =
+		process.env.NODE_ENV === 'production'
```

Comment:
What happens in other environments with sourcepoint?

Reply:
another domain is used: `https://cdn.privacy-mgmt.com`

Reply:
Ah ok. I don't suppose it's a major problem, but `NODE_ENV` will be `'production'` in CODE as well. You may already know, but the value may be interpreted by a number of frameworks and their behaviour changed accordingly. So it's only _not_ `'production'` in DEV - as we want CODE to largely mirror PROD in behaviour.